doc/user/content/sql/functions/to_char.md

```diff
@@ -0,0 +1,110 @@
+| `MM`                        | month number (01-12)                                                                             |
+| `DAY`                       | full upper case day name (blank-padded to 9 chars)                                               |
+| `Day`                       | full capitalized day name (blank-padded to 9 chars)                                              |
+| `day`                       | full lower case day name (blank-padded to 9 chars)                                               |
+| `DY`                        | abbreviated upper case day name (3 chars in English, localized lengths vary)                     |
+| `Dy`                        | abbreviated capitalized day name (3 chars in English, localized lengths vary)                    |
+| `dy`                        | abbreviated lower case day name (3 chars in English, localized lengths vary)                     |
+| `DDD`                       | day of year (001-366)                                                                            |
+| `IDDD`                      | day of ISO 8601 week-numbering year (001-371; day 1 of the year is Monday of the first ISO week) |
+| `DD`                        | day of month (01-31)                                                                             |
+| `D`                         | day of the week, Sunday (1) to Saturday (7)                                                      |
+| `ID`                        | ISO 8601 day of the week, Monday (1) to Sunday (7)                                               |
+| `W`                         | week of month (1-5) (the first week starts on the first day of the month)                        |
+| `WW`                        | week number of year (1-53) (the first week starts on the first day of the year)                  |
+| `IW`                        | week number of ISO 8601 week-numbering year (01-53; the first Thursday of the year is in week 1) |
+| `CC`                        | century (2 digits) (the twenty-first century starts on 2001-01-01)                               |
+| `J`                         | Julian Day (days since November 24, 4714 BC at midnight)                                         |
+| `Q`                         | quarter (ignored by to_date and to_timestamp)                                                    |
+| `RM`                        | month in upper case Roman numerals (I-XII; I=January)                                            |
+| `rm`                        | month in lower case Roman numerals (i-xii; i=January)                                            |
+| `TZ`                        | upper case time-zone name                                                                        |
+| `tz`                        | lower case time-zone name                                                                        |
+
+Specifier Modifiers:
```

Comment:
Maybe make this an `<h3>`? and ensure that the word "specifier" here is in sync with the name elsewhere.

Reply:
Also headers should use sentence case, so "Specifier modifiers"